### PR TITLE
Change directory structure and associated configs.

### PR DESCRIPTION
**Pull request by 7ranceaddic7**
Change to eliminate "unholy characters" (aka spaces) in the directory "Part Bin" to "PartBin".

Changes to configs as required.

Note: AJE has references (Squad.cfg) to VSR and should be notified of implementation. A PR (KSP-RO/AJE#31) has already been submitted for implementation.

### CODE REVIEW

**Comment by Kerbas-ad-astra** [comment]
Thanks for looking into this.  I think there's another mod (Realism Overhaul maybe?) that references VSR models.

**Comment by 7ranceaddic7** [comment]
I'll dig into it.

**Comment by Kerbas-ad-astra** [comment]
I didn't know AJE did, I'm thinking of something that needs one of the Poodle models.  I'll see if I can find it this weekend.

**Comment by DRVeyl** [comment]
I will take care of the RO-side patches making direct reference to VSR models.

File: GameData/VenStockRevamp/PartBin/Data/Tanks.cfg [hunk 1]
```diff
@@ -280,7 +280,8 @@ PART {
 	}
 	MODEL
 	{
-		model = VenStockRevamp/Part Bin/NewParts/Propulsion/fuelTankOscarE
+		model = VenStockRevamp/Part
+		Bin/NewParts/Propulsion/fuelTankOscarE
```
Comment (DRVeyl) [inline GameData/VenStockRevamp/PartBin/Data/Tanks.cfg:284]: Extra CR here

**Comment by 7ranceaddic7** [comment]
CR fix, thanks @DRVeyl

File: GameData/VenStockRevamp/PathPatches/All-PathPatches.cfg [hunk 2]
```diff
@@ -125,11 +125,11 @@
 		@model ^= :^Squad/Parts/Utility/decouplerRadialTT-38K/model:VenStockRevamp/Squad/Parts/Propulsion/TT-38K:
 		@model ^= :^Squad/Parts/Utility/decouplerRadialTT-70/model:VenStockRevamp/Squad/Parts/Propulsion/TT-70:
 		@model ^= :^Squad/Parts/Utility/decouplerSeparatorTR-18D/model:VenStockRevamp/Squad/Parts/Propulsion/TR-18D:
-		@model ^= :^Squad/Parts/Utility/decouplerSeparatorTR-2C/model:VenStockRevamp/Squad/Parts/SmallParts/SmallSeperator:
+		@model ^= :^Squad/Parts/Utility/decouplerSeparatorTR-2C/model:VenStockRevamp/Squad/Parts/PropulsionSmallSeperator:
```
Comment (Kerbas-ad-astra) [inline GameData/VenStockRevamp/PathPatches/All-PathPatches.cfg:128]: Missing a slash here?

Comment (7ranceaddic7) [reply GameData/VenStockRevamp/PathPatches/All-PathPatches.cfg]: Fixed.

**Comment by DRVeyl** [comment]
Wanted to check on this, are any additional changes required?

**Pull request merged**